jekyll/_cci2/persist-data.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: classic-docs
+title: "Persisting Data"
+description: "A guide to the various ways to persist data in CircleCI"
+version:
+- Cloud
+- Server v2.x
+---
+
+This guide gives an overview of the various ways to persist data within and beyond your CircleCI builds. There are a number of ways to move data into, out of and between jobs, and persist data for future use. Using the right feature for the right task will help speed up your builds and improve repeatability and efficiency.
+
+* TOC
+{:toc}
+
+## Caching Strategies
+
+![caching data flow]({{ site.baseurl }}/assets/img/docs/caching-dependencies-overview.png)
+
+Caching persists data between the same job in different Workflow builds, allowing you to reuse the data from expensive fetch operations from previous jobs. After an initial job run, future instances will run faster as they will not need to redo the work. A prime example is package dependency managers such as Yarn, Bundler, or Pip. With dependencies restored from a cache, commands like yarn install will only need to download new dependencies, if any, and not redownload everything on every build.
+
+Caches are global within a project. A cache saved on one branch will be used by jobs run on other branches so they should only be used for data that is suitable to share across branches.
+
+**Caches created via the save_cache step are stored for up to 15 days.**
+
+For more information see the [Caching Dependencies]({{site.baseurl}}/2.0/caching/) guide.
+
+## Using Workspaces
+
+![workspaces data flow]( {{ site.baseurl }}/assets/img/docs/workspaces.png)
+
+When a workspace is declared in a job, files and directories can be added to it. Each addition creates a new layer in the workspace filesystem. Downstream jobs can then use this workspace for their own needs or add more layers on top.
+
+Unlike caching, workspaces are not shared between runs as they no longer exists once a workflow is complete. The one exception is when workflows are rerun.
+
+**Workspaces are stored for up to 15 days.**
```

Comment:
The line prior to this one states that a workspace no longer exists once a workflow is complete. If they are stored for up to 15 days... does that mean, only in the context of "rerunning a workflow" is that data made available?

Reply:
Yeah good question - I think that's correct but will check with eng

Reply:
https://circleci.slack.com/archives/CNPG1R4QZ/p1601650871053200